TODOLIST.txt

```diff
@@ -0,0 +1,40 @@
+TODOLIST
+--------
+
+If you want a really fast Drupal 8:
+
+ * Drupal\Core\KeyValueStore\DatabaseStorage
```

Comment:
Also note that there is DatabaseStorageExpirable, which should be easier to use on most sites, because you can't just switch backend, you need to migrate your data.. for most (all) expirable things, that shouldn't be necessary.